README.md

```diff
@@ -156,6 +156,21 @@ Then, navigate to the root directory of the project and run:
 pip install --editable '.[all]'
 ```
 
+###For Windows machine
+
+Install Conda use Windows exe from Conda Downloads website:
+https://docs.conda.io/projects/miniconda/en/latest/
+
+Add PATH to Environment Variables
+Sample value to add:  C:\ProgramData\miniconda3\condabin
+
+To avoid error "Unable to determine R Home:
+- Install R
+- Create R_HOME system variable
+- Add R_HOME\bin to the PATH, in order to execute R from python
+- Add R_HOME\bin\x64 to the PATH, in order to load R.dll
+- Install package tzlocal
```

Comment:
nit: perhaps add a list of command lines to execute for this? That would help readers save a lot of time!